Source/WebKit/UIProcess/ios/WKActionSheet.mm

```diff
@@ -216,15 +216,11 @@ - (void)updateSheetPosition
     // - The completion of the view controller dismissal in willRotate.
     // (We cannot present something again until the dismissal is done)
 
-    BOOL isBeingPresented = [presentedViewController presentingViewController] || [self presentingViewController];
```

Comment:
I'm pretty sure this is a separate local variable on purpose, to clarify the intended purpose of this check without requiring a comment. Let's either keep it as-is, or make a static helper.

Reply:
Not true, the comment above says "we cannot present something again until the dismissal is done" which means as long as the presentingViewController is active, nothing can be done.